types/index.d.ts

```diff
@@ -0,0 +1,160 @@
+import type { Client, Subscription } from 'aedes';
+import type { AedesPacket } from 'aedes-packet';
+import type { QoS } from 'mqtt-packet';
+import type { Readable } from 'stream';
+
+export type { AedesPacket as Packet } from 'aedes-packet';
+
+export interface AedesPersistenceSubscription {
+  clientId: string;
+  topic: string,
+  qos?: QoS
+}
+
+type LastHearthbeatTimestamp = Date;
+
+export interface Brokers {
+  [brokerId: string]: LastHearthbeatTimestamp;
+}
```

Comment:
why this? Shouldn't this be in aedes itself?

Reply:
Yes but like mentioned in a conversation above with @robertsLando , it isn't currently. A PR on Aedes to add Brokers interface is scheduled.

Reply:
I would prefer to have a PR to Aedes first and then reuse that here.